### PR TITLE
[WIP] Full audit: RBAC permissions matrix for Cloud Databases

### PR DESCRIPTION
**Pull request by DonNateR**
- Fixes issue #1620 

-Change table to RBAC standard

### CODE REVIEW

**Comment by rackernexus** [comment]
HULK [PREVIEW](https://staging.developer.rackspace.com/support.rackspace.com/build-9e48571a8f/how-to/)

**Comment by kmsholcomb** [comment]
Hi, @DonNateR. My markup for the article is in the attached docs. I suggested other changes based on the template for these articles that Stephanie created before the break. Please let me know if you have any questions. Thanks!

[Permissions matrix for Cloud Databases_editsKH.pdf](https://github.com/rackerlabs/rackspace-how-to/files/683104/Permissions.matrix.for.Cloud.Databases_editsKH.pdf)
[Permissions matrix for Cloud Databases_editsKH.docx](https://github.com/rackerlabs/rackspace-how-to/files/683103/Permissions.matrix.for.Cloud.Databases_editsKH.docx)

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/support.rackspace.com/build-dd2f84f648/how-to/). :bow:

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/support.rackspace.com/build-de45d7a735/how-to/).

File: content/cloud-databases/permissions-matrix-for-cloud-databases.md [hunk 1]
```diff
@@ -1,189 +1,168 @@
...
-| List flavors for the datastore version | `GET /datastores/{datastoreType}/versions/{versionId}/flavors` |	**Observer & Creator & Admin** |			Lists flavors for a datastore version. |
+| List flavors |	`GET /{version}/{accountId}/flavors` |	**Admin, Creator, Observer** |	Lists information for all available flavors. |
+| List flavors by ID |	`GET /{version}/{accountId}/flavors/{flavorId}`	 |	**Admin, Creator, Observer** |	Lists information about the specified flavor. |
+| List flavors for the datastore version | `GET /{version}/{accountId}/datastores/{datastoreType}/versions/{versionId}/flavors` |	**Admin, Creator, Observer** | Lists flavors for a datastore version. |
 
-### On Demand Backups
+### On-demand backups
 
-**Note**: Any user calling the on demand backup operations for Cloud Databases must have access to Cloud Files.
+**Note**: Any user who calls the on demand backup operations for Cloud Databases must have access to Cloud Files.
```
Comment (catlook) [inline content/cloud-databases/permissions-matrix-for-cloud-databases.md]: ... on-demand ...

File: content/cloud-databases/permissions-matrix-for-cloud-databases.md [hunk 2]
```diff
@@ -1,189 +1,168 @@
...
+| Create a backup |	`POST /{version}/{accountId}/backups`	| **Admin, Creator** |	Creates a new backup for a database instance. |
+| Delete a backup|	`DELETE /{version}/{accountId}/backups/{backupId}` |	**Admin** | Deletes the specified backup. |
+| List backups |	`GET /{version}/{accountId}/backups` |	**Admin, Creator, Observer** | Lists all backups for all database instances. |
+| List backup by ID |	`GET /{version}/{accountId}/backups/{backupId}` |	**Admin, Creator, Observer** | Lists details about the specified backup. |
+| List backups for instance |	`GET /{version}/{accountId}/instance/{instanceId}/backups` | **Admin, Creator, Observer** | Lists all backups for the specified instance.
+| Restore a backup |	`POST /{version}/{accountId}/instances` |	 **Admin, Creator**	| Creates a new database instance from a backup. |
 
 ### Scheduled Backups
 
 **Note:** Any user calling the scheduled backup operations for Cloud Databases must have access to Cloud Files.
```
Comment (catlook) [inline content/cloud-databases/permissions-matrix-for-cloud-databases.md]: ... user who calls on ...

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-123bada7c6/how-to/)

**Pull request merged**